lib/app/service/installer.go

```diff
@@ -455,5 +455,5 @@ var uploadScriptTemplate = template.Must(template.New("uploadScript").Parse(`#!/
 #
 #    http://www.apache.org/licenses/LICENSE-2.0
 #
-./gravity --insecure update upload --state-dir=.
+./gravity --insecure update upload
```

Comment:
Will this work if the upgrade/upload script invoked from some directory other than unpacked tarball? E.g.
```
$ cd ~
$ sudo ~/installer/upload
```

Reply:
This won't and it did not work before as well. Is this a requirement? The script will error on being unable to find the `gravity` binary (if there's none in the $HOME directory) or might error (subject to version of the binary from home directory) on being unable to find installer state if there's a binary directly in $HOME.